### PR TITLE
MatlabIndentationBear: Add asciinema

### LINKED ISSUE #1221: MatlabIndentationBear.py: Add asciinema
The coala bear MatlabIndentationBear.py does not have an asciinema.

### PR DESCRIPTION
**Pull request by Elgayar**
MatlabIndentationlBear: Add asciinema

Closes #1221

### CODE REVIEW

**Comment by Elgayar** [comment]
I don't quite get you, can you please give more detail? @Nosferatul

**Comment by Elgayar** [comment]
Done! Edited the commit message and the pull request title and body to capitalize the word Add!
@Nosferatul @vijeth-aradhya

**Comment by Makman2** [comment]
1. Why Matlab***International***Bear? :O It's Matlab**Indentation**Bear ;)
2. you repeat the shortlog in the commit message body again, no need for that^^

**Comment by sils** [comment]
:D I didn't know we had international bears

**Comment by Elgayar** [comment]
I must've been dreaming or something while typing that commit xD @sils @Makman2 
Anyways I edited the 3 things you asked for!

**Comment by sils** [comment]
Hi,

it looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.* You already seem to have done it but don't worry, the rebase will get rid of the merge commit for you.)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala/coala-bears.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix it up for you - just tell us :).

Ping us on [our gitter channel](https://coala.io/chat) if you need help!

File: bears/matlab/MatlabIndentationBear.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ class MatlabIndentationBear(LocalBear):
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/97596'
```
Comment (sils) [inline bears/matlab/MatlabIndentationBear.py]: hey, you're running the bear on a python file, not on the matlab file. The python file with the tests contains code samples, for a proper asciinema you'll have to take the matlab code samples and put them in `.m` files, run the bear on those.

Let us know if you need any help with that!

**Comment by jayvdb** [comment]
GCI task which wasnt completed.

**Pull request closed without merging**